src/lib/viewers/BaseViewer.js

```diff
@@ -1037,6 +1040,25 @@ class BaseViewer extends EventEmitter {
         this.annotator.scrollToAnnotation(data);
     }
 
+    /**
+     * Handles retrieving the active annotation id from a deep-linked annotation and scrolls to the annotation.
+     * @param {Object[]} data - annotations array from emitted from box-annotations.
+     * @return {void}
+     */
+    handleAnnotationsOnLoad = (data = []) => {
+        const {
+            file: { id },
+        } = this.options;
+
+        const loadedAnnotationId = getProp(this.options, `fileOptions.${id}.annotations.activeId`, null);
+
+        if (!loadedAnnotationId) {
```

Comment:
What does `loaded` mean in this case? Should it be `activeAnnotationId`?

Reply:
Loaded meaning it was the activeAnnotationId on load... but I can re-name it.